eo-runtime/src/main/java/EOorg/EOeolang/EOrust.java

```diff
@@ -128,9 +134,9 @@ public EOrust(final Phi sigma) {
                             "EOrust.natives.%s",
                             name
                         )
-                    ).getDeclaredMethod(name, new Class[]{EOrust.class});
-                    return new Data.ToPhi(
-                        Long.valueOf((int) method.invoke(null, this))
+                    ).getDeclaredMethod(name, EOrust.class);
+                    return EOrust.translate(
+                        (byte[]) method.invoke(null, this)
```

Comment:
@levBagryansky Can we handle this type casting with some error throwing? Or can we redesign it, according to: https://www.yegor256.com/2015/04/02/class-casting-is-anti-pattern.html

Reply:
@Graur what about such validation?